src/components/Text/index.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import styled from 'styled-components';
+import { HTMLAttributes } from 'react';
+
+type HeadingAsTagProps = {
+  asTag?: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
+};
+
+type TextProps = {
+  size?: 'xxl' | 'xl' | 'l' | 'm' | 's' | 'xs' | 'xxs';
```

Comment:
Maybe it's useful to also have the option to make the color secondary? (this is something we do quite often).
If you don't like it, atleast we should pass the classname so we can modify it with a styled component

Reply:
The classname is already passed, I'll see what I can do about the colors.